deploy-help/delete-database.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash -e
+
```

Comment:
I think this script should be merged with `deploy.sh`. `deploy.sh` should have an option that tells it whether to migrate or destroy the database before deployment. Migration should be the default.